src/components/wallet/PageTransactions.vue

```diff
@@ -216,7 +214,7 @@ export default {
       },
       update(result) {
         this.dataLoaded = true
-        this.lastLoadedRecordsCount = result.transactionsV2.length
+        this.moreAvailable = result.transactionsV2.length > 0
```

Comment:
same here, would rap it with `( ... )` for evaluation

Reply:
```suggestion
        this.moreAvailable = (result.transactionsV2.length > 0)
```